pkg/config/cliconfig.go

```diff
@@ -15,13 +15,22 @@ import (
 	"github.com/docker/cli/cli/config/types"
 )
 
-var (
-	darwinHelpers  = []string{"osxkeychain", "file"}
-	windowsHelpers = []string{"wincred", "file"}
-	linuxHelpers   = []string{"secretservice", "pass", "file"}
+const (
+	Wincred       = "wincred"
+	Osxkeychain   = "osxkeychain"
+	Secretservice = "secretservice"
+	Pass          = "pass"
+	File          = "file"
+	Sqlite        = "sqlite"
```

Comment:
I think appending `CredHelper` to each of these would give them a more descriptive name.

Reply:
Fixed